pythonforandroid/recipes/ffmpeg/__init__.py

```diff
@@ -52,7 +51,7 @@ def build_arch(self, arch):
                 build_dir = Recipe.get_recipe(
                     'libx264', self.ctx).get_build_dir(arch.arch)
                 cflags += ['-I' + build_dir + '/include/']
-                ldflags += ['-lx264', '-L' + build_dir + '/lib/']
+                ldflags += [build_dir + '/lib/' + 'libx264.a']
```

Comment:
Recent versions of ffmpeg seem to prioritize searching for a dynamic library, ignoring the static one unless explicitly defined. The solution, then, is to provide the path to the static library directly.

Reply:
Great findings, do you mind putting this exact comment above that line in the source?

Reply:
Good ideia, done! 👍